### ISSUE TITLE
Safari iOS 15.4.1 lazy loading not working.

### ISSUE THREAD

**Issue by AleMadLei** [dinbror/blazy#186]
Images sometimes load and sometimes they won't.

This seems related to the following https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes#HTML, added native lazy loading.

This works well on older iOS safari versions. Just not the latest.

**Comment by invasi0n** [comment]
Hello!
Do you have a solution for this problem?
Thank you~

**Comment by AleMadLei** [comment]
Nope, on the project I was using it we basically removed the library and use the default lazy loading from the browser.
